pyproject.toml

```diff
@@ -10,7 +10,7 @@ dev = [
     "pytest>=8.0.0",
     "pytest-cov>=6.0.0",
     "ruff>=0.8.0",
```

Comment:
Do we really need ruff?

Reply:
Sorry, I don't quite understand the question; we don't need ruff. 

It's just being used as a linter/formatter for the few python files, is there any particular reason to not use it or have it?

Reply:
Oh, my mistake, I thought it was in the regular dependencies, not in dev.

Reply:
No worries! Thanks for looking!